src/components/Speciality/Projects.js

```diff
@@ -52,7 +52,7 @@ const CompaniesHelped = ({ clients, noOther }) => (
 const ProjectsSection = ({ related, title, clients }) => (
   <Grid>
     <Padding top={5} bottom={5}>
-      {related ? (
+      {!!(related && related.length) && (
         <Row>
```

Comment:
You don't need the `!!` here.

Reply:
It's preventative to avoid this: https://github.com/yldio/yld.io/pull/278#discussion_r299540859 !